gms/feature_service.cc

```diff
@@ -41,6 +41,10 @@ feature_config::feature_config() {
 }
 
 feature_service::feature_service(feature_config cfg) : _config(cfg) {
+#ifdef SCYLLA_ENABLE_ERROR_INJECTION
+    get_suppressed_features();
```

Comment:
Nit: `get_` suggests that this is some kind of a getter and returns something. I suggest changing it e.g. to `initialize_suppressed_features()`. Alternatively:

```c++
_suppressed_features = get_suppressed_features();
```

... while changing `get_suppressed_features()` into a free function (i.e. not a method).